camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelSimpleAnnotator.java

```diff
@@ -46,7 +46,7 @@ boolean isEnabled() {
      * if the expression is not valid a error annotation is created and highlight the invalid value.
      */
     void validateText(@NotNull PsiElement element, @NotNull AnnotationHolder holder, @NotNull String text) {
-        boolean hasSimple = text.contains("${") || text.contains("$simple{");
+        boolean hasSimple = text.contains("$simple{");
```

Comment:
That should be `${` or `$simple{` as it was before. The idea is to only parse/validate the simple text if there is an embedded funktion which must start with `${` or `$simple{`. If there is no embedded functions then its just regular text and no parsing/validation is needed.